support-frontend/assets/components/directDebit/directDebitProgressiveDisclosure/directDebitForm.tsx

```diff
@@ -17,7 +17,7 @@ import {
 	setRecaptchaToken,
 } from 'helpers/redux/checkout/recaptcha/actions';
 import type { CheckoutState } from 'helpers/subscriptionsForms/subscriptionCheckoutReducer';
-import { nonSillyCharacters } from 'helpers/subscriptionsForms/validation';
+import { zuoraCompatibleString } from 'helpers/subscriptionsForms/validation';
```

Comment:
I’ve renamed this to be more accurate, especially because some of the characters that Zuora can’t handle (and that we are now blocking with this validation) are perfectly valid characters (e.g. to have in a name), we just don’t support them.